tutorials/dataframe/df102_NanoAODDimuonAnalysis.C

```diff
@@ -0,0 +1,122 @@
+/// \file
+/// \ingroup tutorial_dataframe
+/// \notebook -nodraw
+/// This tutorial illustrates how NanoAOD files can be processed with ROOT
+/// dataframes. The NanoAOD-like input file is filled with events from
+/// CMS OpenData containing muon candidates from 2011 data. The script matches
+/// muon pairs and produces an histogram of the dimuon mass spectrum showing
+/// particle resonances up to the Z mass.
+///
+/// \macro_code
+///
+/// \date August 2018
+/// \author Stefan Wunsch
+
+#include "ROOT/RDataFrame.hxx"
+#include "ROOT/RVec.hxx"
+#include "TCanvas.h"
+#include "TH1D.h"
+#include "TLatex.h"
+#include "TLorentzVector.h"
+#include "TStyle.h"
+
+using namespace ROOT::VecOps;
+
+template <typename T>
+void Draw(T h_result_ptr);
```

Comment:
it's convenient and i didn't know the type by heart ;)

Reply:
i can replace it by the type. then it is more transparent to the user.

Reply:
I would not shoot a template at this since we know the type. Maybe we can pass by reference a histo and draw it to show it is just a plotter function as the millions every users writes during her phd?

Reply:
fixed, put the `GetValue` not in the plotting function